hmftools.wdl

```diff
@@ -0,0 +1,927 @@
+        purpleSvVcfIndex: {description: "The index of the structural variants VCF file produced by purple.", category: "required"}
+        purpleSomaticVcf: {description: "The VCF file produced by purple which contains somatic variants.", category: "required"}
+        purpleSomaticVcfIndex: {description: "The index of the somatic VCF file produced by purple.", category: "required"}
+        outputDir: {description: "The directory the ouput will be placed in.", category: "common"}
+        memory: {description: "The amount of memory this job will use.", category: "advanced"}
+        javaXmx: {description: "The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.",
+                  category: "advanced"}
+        timeMinutes: {description: "The maximum amount of time the job will run in minutes.", category: "advanced"}
+        dockerImage: {description: "The docker image used for this task. Changing this may result in errors which the developers may choose not to address.",
+                      category: "advanced"}
+    }
+}
+
+task CuppaChart {
+    input {
+        String sampleName
+        File cupData
+        String outputDir = "./cuppa"
+
+        String memory = "4G"
+        Int timeMinutes = 5
+        String dockerImage = "quay.io/biowdl/cuppa:1.4"
+    }
+
+    command {
+        set -e 
+        mkdir -p ~{outputDir}
+        cuppa-chart \
+        -sample ~{sampleName} \
+        -sample_data ~{cupData} \
+        -output_dir ~{outputDir}
+    }
+
+    output {
+        File cuppaChart = "~{outputDir}/~{sampleName}.cuppa.chart.png"
+        File cuppaConclusion = "~{outputDir}/~{sampleName}.cuppa.conclusion.txt"
+    }
+
+    runtime {
+        memory: memory
+        time_minutes: timeMinutes # !UnknownRuntimeKey
+        docker: dockerImage
+    }
+
+    parameter_meta {
+        sampleName: {description: "The name of the sample.", category:"common"}
+        cupData: {description: "The cuppa output.", category: "required"}
+        outputDir: {description: "The directory the output will be written to.", category:"common"}
+        memory: {description: "The amount of memory this job will use.", category: "advanced"}
+        timeMinutes: {description: "The maximum amount of time the job will run in minutes.", category: "advanced"}
+        dockerImage: {description: "The docker image used for this task. Changing this may result in errors which the developers may choose not to address.",
+                      category: "advanced"}
+    }
+}
+
+task GripssApplicationKt {
+    input {
+        File inputVcf
+        String outputPath = "gripss.vcf.gz"
+        String tumorName
+        String normalName
+        File referenceFasta
+        File referenceFastaFai
+        File referenceFastaDict
+        File breakpointHotspot
+        File breakendPon
+        File breakpointPon
+
+        String memory = "33G"
+        String javaXmx = "32G"
```

Comment:
Can these be decreased by 1? The application will probably still be able to run, and 4 jobs can be scheduled on a 128 GB node instead of 3.